apps/web/src/components/search/Highlight.tsx

```
@@ -0,0 +1,15 @@
+import { FC } from "react";
+import { searchQueryAtom } from "./state";
+
+export const Highlight: FC<{ text: string }> = ({ text }) => {
+  const searchTerm = useAtomValue(searchQueryAtom);
+
+  const highlightedText = oramaMatchHighlighter.highlight(
+    text,
+    searchTerm,
+  ).HTML;
+
+  return <span dangerouslySetInnerHTML={{ __html: highlightedText }} />;
```

Biome (2.1.2)

[error] Avoid passing content using the dangerouslySetInnerHTML prop.

Setting content using code can expose users to cross-site scripting (XSS) attacks

(lint/security/noDangerouslySetInnerHtml)